### DISCUSSION TITLE
unable to use custom domain for pages

### DISCUSSION THREAD

**Discussion by edvorg** [community/community#29563] [Other Feature Feedback, Questions, & Ideas] [answered]
hi I'm getting this error, I tried following troubleshooting section, but no luck

Both pawsitive.care and its alternate name are improperly configured
Domain does not resolve to the GitHub Pages server. For more information, see [documentation](https://docs.github.com/articles/setting-up-a-custom-domain-with-github-pages/) (NotServedByPagesError).

**Comment by mazkdevf** [comment]
Hello edvorg!

What is the error?

**Reply by edvorg** [reply]
hello, my apologies for submitting this, I just had to wait for some time, I checked next morning and it's working perfectly fine now

**Comment by airtower-luna** [comment, marked as answer]
How long has it been since you configured the DNS entries for your domain? They look correct to me at a glance, but might take a while to propagate through caches.

**Reply by edvorg** [reply]
hello, my apologies for submitting this, I indeed just had to wait for some time, I checked next morning and it's working perfectly fine now
